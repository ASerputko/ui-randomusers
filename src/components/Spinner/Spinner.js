import React from 'react';

/**
 * Show spinner
 */
const Spinner = () => {
  return (
    <div style={container}>Loading....</div>
  );
};

const styles = {
  container: {
    display: 'flex',
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
    fontSize: 20,
    fontWight: 'bold'
  }
};

const { container } = styles;

export default Spinner;
