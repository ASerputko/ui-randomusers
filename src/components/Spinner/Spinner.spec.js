import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './Spinner';

describe('Spinner', () => {
  it('should render spinner ', () => {
    const tree = renderer
      .create(<Spinner />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
