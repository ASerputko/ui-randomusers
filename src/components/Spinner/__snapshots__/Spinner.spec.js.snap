// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render spinner  1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "row",
      "fontSize": 30,
      "fontWight": "bold",
      "justifyContent": "center",
      "marginTop": 50,
    }
  }
>
  Loading....
</div>
`;
